touchforms/formplayer/static/formplayer/script/fullform-ui.js

```diff
@@ -367,6 +367,17 @@ Formplayer.ViewModels.EvaluateXPath = function() {
     };
 }
 
+Formplayer.ViewModels.SyncUserData = function() {
+    var self = this;
+    self.evaluate = function(form) {
+        var callback = function(result, status) {
+            self.result(result);
+            self.success(status === "success");
+        };
+        $.publish('formplayer.sync-user-data', [self.xpath(), callback]);
```

Comment:
also we'll probably need to unsubscribe from this in teardown. can talk to you abut that